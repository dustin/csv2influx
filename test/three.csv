name,time,sensor,site,value
env,1599612130709842134,thing,site1,2
env,1599612130743049304,thing,site1,3
env,1599612306307120495,thing,site1,5
