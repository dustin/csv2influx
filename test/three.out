env,site=site1,sensor=thing value=2i 1599612130709842134
env,site=site1,sensor=thing value=3i 1599612130743049304
env,site=site1,sensor=thing value=5i 1599612306307120495
